README.md

```diff
@@ -7,8 +7,8 @@ These packages are for the Arch linux ARM variant of Asahi Linux.
 First import the signing key
 
 ```bash
-sudo pacman-key --recv-keys 1F35A37F9D85CBC2B74F595E37CE6162C969C50E
-sudo pacman-key --lsign-key 1F35A37F9D85CBC2B74F595E37CE6162C969C50E
+sudo pacman-key --recv-keys 12CE6799A94A3F1B5DDFFE88F576553597FB8FEB
```

Comment:
how will this work? Will we both have the private key for this?

Reply:
oh I just saw the secrets repo 👍

Reply:
so will need to make a new release signed with that key. Also we'll have to update the `asahi-alarm-keyring` repo

Reply:
@joske 
- https://github.com/asahi-alarm/asahi-alarm-keyring/pull/1